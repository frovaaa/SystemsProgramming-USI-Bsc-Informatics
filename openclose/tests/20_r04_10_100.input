09Aa16
2F38fHh
97926
88222
J5575j
85304
36658
33947
77933
L8613l
3Dd380
Bb0701
Gg0K6kYy
80253
399V9v
23339
03550
016Ff1
60025
80167
64285
287AaIi
11058
6F617f
711T2t
09487
71977
4865Bb
20P29p
058K0k
28PCc4p
07310
W8050w
67600
45588
47748
03Rr54
61402
38825
48885
35707
96813
78490
81874
780Pp2
41239
42392
56175
73365
11430
54297
1056Qq
166N1n
0S079s
32033
09546
3084Vv
41002
5V750v
1749Jj
03705
A1572a
823F2f
45283
50698
Gg9891
16452
01912
8892Oo
84B96b
52242
26802
01372
05583
75141
13917
63628
57Nn70
23881
619G2g
90839
PO3X3xop
24675
8A206a
32604
42346
8Mm76Cc
1712Hh
55384
59969
4Ee880
56342
0825Dd
77873
28428
8314Ii
21135
G6620g
372D8d
40196
