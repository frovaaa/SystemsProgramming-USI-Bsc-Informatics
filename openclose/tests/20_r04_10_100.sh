#!/bin/bash
exec "$1" "AaBbCcDdEeFfGgJjIiHhKkLlMmNnOoPpQqRrSsTtUuVvWwXxYyZz" < tests/20_r04_10_100.input
