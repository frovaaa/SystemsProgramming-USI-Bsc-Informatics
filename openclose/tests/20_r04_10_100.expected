0916
2F38f
97926
88222
J5575j
85304
36658
33947
77933
L8613l
3380
0701
0K6k
80253
399V9v
23339
03550
0161
60025
80167
64285
287
11058
6F617f
711T2t
09487
71977
4865
20P29p
058K0k
28P4p
07310
W8050w
67600
45588
47748
0354
61402
38825
48885
35707
96813
78490
81874
7802
41239
42392
56175
73365
11430
54297
1056
166N1n
0S079s
32033
09546
3084
41002
5V750v
1749
03705
A1572a
823F2f
45283
50698
9891
16452
01912
8892
84B96b
52242
26802
01372
05583
75141
13917
63628
5770
23881
619G2g
90839
PO3X3xop
24675
8A206a
32604
42346
876
1712
55384
59969
4880
56342
0825
77873
28428
8314
21135
G6620g
372D8d
40196
